### ISSUE TITLE
Follow option for renamed files

### ISSUE THREAD

**Issue by garbagerodin** [gitextensions/gitextensions#6274]
## Steps to reproduce

0) Enable option "Follow renames in file history" in Settings => Git Extensions => General
1) Commit file
2) Change file
3) Rename file
4) Change file
5) Show file history

## Current behaviour

History show commits only after file rename.

## Expected behaviour

History show full history with commits before file rename.

**Console "git log --follow" work correctly and show full history.**

## Screenshots

![commits](https://user-images.githubusercontent.com/40666182/53010756-c2657300-344f-11e9-865f-80ff2ef0b74a.png)

![history](https://user-images.githubusercontent.com/40666182/53010765-c72a2700-344f-11e9-84e2-8cdc1ff6b157.png)

## Did this work in previous version of GitExtensions

No. Checked 3.00.00.4433.

## Environment

- Git Extensions 3.0.2.5232
- Build 23b6f51905006ccdde8701591df706284b4155dc
- Git 2.20.1.windows.1
- Microsoft Windows NT 10.0.17134.0
- .NET Framework 4.7.3260.0
- DPI 96dpi (no scaling)

## Diagnostics

"git log --follow"
[git_log.txt](https://github.com/gitextensions/gitextensions/files/2879706/git_log.txt)

Git command log  attached
[CommandLog.txt](https://github.com/gitextensions/gitextensions/files/2879682/CommandLog.txt)

**Comment by vbjay** [comment]
Search for issues next time.

**Comment by RussKie** [comment]
Duplicate of #5944
